### PR TITLE
First pass at layout of landing page

### PR DESCRIPTION
**Pull request by creuter**
screencapture-localhost-4000-1458330754782
- Add autoprefixer, for vendor prefixes
- Add a separate layout for the landing page
- Change `back-link` to `appended-links`, because they aren't always links back.

### CODE REVIEW

File: web/controllers/page_controller.ex [hunk 1]
```diff
@@ -2,6 +2,7 @@ defmodule Community.PageController do
   use Community.Web, :controller
 
   def index(conn, _params) do
-    render conn, "index.html"
+    render conn, "index.html",
```
Comment (drapergeek) [inline web/controllers/page_controller.ex]: How about:

``` elixir
render(
  conn,
  "index.html",
  layout: {Community.LayoutView, "landing.html"}
)
```

File: web/templates/job/new.html.eex [hunk 2]
```diff
@@ -5,7 +5,10 @@
     you'll be sent an email asking if you'd like to extend (for&nbsp;free).</p>
     <p>Your position should be near Raleigh, NC, and related to UX, UI, or
     visual design for applications or the&nbsp;web.</p>
-    <%= link("← Back to Job Board", to: job_path(@conn, :index), class: "back-link") %>
+    <%= link("← Back to Job Board", to:
```
Comment (drapergeek) [inline web/templates/job/new.html.eex]: How about bringing the `to:` down on a new line and putting the closing `) %>` on is own line?

File: web/static/css/_landing-header.scss [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  background-image: url("/images/leaves.svg");
+  background-position: 50%;
+  background-repeat: no-repeat;
+  border-bottom: $base-border;
+  margin-bottom: $large-spacing;
+  text-align: center;
+}
+
+.landing-header-title {
+  margin-bottom: $large-spacing * 2;
```
Comment (kylefiedler) [inline web/static/css/_landing-header.scss]: Maybe `$x-large-spacing`?

File: web/static/css/_site-footer.scss [hunk 4]
```diff
@@ -0,0 +1,24 @@
+.site-footer {
+  @include padding($large-spacing null);
```
Comment (kylefiedler) [inline web/static/css/_site-footer.scss:2]: Do you need the function here?

**Comment by kylefiedler** [comment]
Couple small things. Maybe call to action up top? I like the style of the header typeface but I don't think it's designed very well, maybe try some others?

**Comment by creuter** [comment]
Thanks @kylefiedler. Any typeface recommendations?

File: web/static/css/_site-footer.scss (hunk 4 above)
Comment (creuter) [inline web/static/css/_site-footer.scss:2]: Would you use two declarations?

**Comment by kylefiedler** [comment]
@creuter what about: https://www.google.com/fonts/specimen/Permanent+Marker

**Pull request merged**